introduction.md

```diff
@@ -1,11 +1,4 @@
-Matt O'Toole
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?